lib/commons/color/color.js

```diff
@@ -150,6 +150,54 @@ export default class Color {
 
     return 0.2126 * r + 0.7152 * g + 0.0722 * b;
   }
+
+  /**
+   * Add the value to a color and return a new instance of the Color. The resulting color values are not clamped to valid values of the color space (must be done separately).
+   * @method add
+   * @memberof axe.commons.color.Color
+   * @instance
+   * @return {Color}
+   */
+  add(value) {
+    return new Color(
+      this.red + value,
+      this.green + value,
+      this.blue + value,
+      this.alpha
+    );
+  }
+
+  /**
+   * Divide a color by the value and return a new instance of the Color
+   * @method divide
+   * @memberof axe.commons.color.Color
+   * @instance
+   * @return {Color}
+   */
+  divide(value) {
```

Comment:
I don't think it's a good idea for us to start having colors that are sometimes 8 bit and other times fractions. That's bound to lead to bugs. See below for more thoughts on that.